lint-tasks.yml

```
@@ -0,0 +1,97 @@
+version: "3"
+
+vars:
+  G_LINT_VENV_DIR: "{{.G_BUILD_DIR}}/lint-venv"
+
+tasks:
+  check:
+    cmds:
+      - task: "cpp-check"
+      - task: "yml-check"
+
+  fix:
+    cmds:
+      - task: "cpp-fix"
+      - task: "yml-fix"
+
+  cpp-check:
+    sources: &cpp_source_files
+      - "{{.TASKFILE}}"
+      - ".clang-format"
+      - "src/**/*.cpp"
+      - "src/**/*.h"
+      - "src/**/*.hpp"
+      - "src/**/*.tpp"
+      - "examples/**/*.cpp"
+      - "examples/**/*.h"
+      - "examples/**/*.hpp"
+      - "examples/**/*.tpp"
+    cmds:
+      - task: "cpp"
+        vars:
+          FLAGS: "--dry-run"
+
+  cpp-fix:
+    sources: *cpp_source_files
+    cmds:
+      - task: "cpp"
+        vars:
+          FLAGS: "-i"
+
+  yml:
+    aliases:
+      - "yml-check"
+      - "yml-fix"
+    deps: ["venv"]
+    cmds:
+      - |-
+        . "{{.G_LINT_VENV_DIR}}/bin/activate"
+        yamllint --strict \
+          .github \
+          .yamllint.yml \
+          .clang-format \
+          docs \
+          lint-tasks.yml \
+          Taskfile.yml
```

```suggestion
        yamllint \
          --config-file "{{.ROOT_DIR}}/tools/yscope-dev-utils/lint-configs/.yamllint.yml" \
          --strict \
          .github \
          .yamllint.yml \
          .clang-format \
          .clang-tidy \
          lint-tasks.yml \
          Taskfile.yml
```

* Added the line to use the `yscope-dev-utils` config file.
* Added `.clang-tidy` to the list of linted files (you'll need to update it to pass the linter).